cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -26,6 +28,8 @@ const Modules = () => {
   const [allData, setAllData] = useState([]);
   const [filteredData, setFilteredData] = useState([]);
   const [loadingModules, setLoadingModules] = useState(false);
+  const [moduleHealthFilter, setModuleHealthFilter] = useState<string[]>([]);
```

Comment:
You should start with all health filters enabled

```suggestion
  const [moduleHealthFilter, setModuleHealthFilter] = useState<string[]>([
    "healthy",
    "unhealthy",
    "unknown",
  ]);
```